### ISSUE TITLE
Typescript Types support

### ISSUE THREAD

**Issue by ghost** [radonjs/Radon#54]
Is typescript definitions something that could be accepted as a PR into this be repository?

**Comment by G0ldenSp00n-zz** [comment]
Yes, as long as it is setup in a way to allow both vanilla JavaScript and TyoeScript as options
